WORKSPACE

```diff
@@ -533,6 +533,8 @@ register_toolchains(
 # starting from 0.29, rules_go requires bazel >= 4.2.0
 http_archive(
     name = "io_bazel_rules_go",
+    patch_args = ["-p1"],
+    patches = ["//:rules_go_integration_testing.patch"],
```

Comment:
Please add a comment why this is needed and also pointing out that it is not needed for downstream users of rules_haskell.

Reply:
Done